### DISCUSSION TITLE
Isar 3 Stable release date.

### DISCUSSION THREAD

**Discussion by muddassar-lab** [isar/isar#616] [Help] [answered]
I want to know when Isar 3 is going to be stable.

**Comment by mikekosulin** [comment, marked as answer]
@leisim said: "Soon 😅 sorry no exact date but very soon. Just a few small fixes, inspector support for the newest features and web support" [Source: 30/08/2022 @ Telegram](https://t.me/isardb/2766)

**Comment by muddassar-lab** [comment]
Thanks Sir For Your Reply.
